Assets/Plugins/CountlySDK/Countly.cs

```diff
@@ -178,6 +178,10 @@ public async void Init(CountlyConfiguration configuration)
 
             Configuration = configuration;
 
+            if (Configuration.EnableFirstAppLaunchSegment) {
+                Debug.LogWarning("'EnableFirstAppLaunchSegment' has been deprecated and this only acts as a dummy.");
```

Comment:
"'EnableFirstAppLaunchSegment' has been deprecated and it's functionality has been removed. This variable is only left for compatability."